enterprise_subsidy/apps/api/paginators.py

```diff
@@ -55,18 +55,20 @@ class SubsidyListPaginator(pagination.PageNumberPagination):
 
     def __init__(self, *args, **kwargs):
         super().__init__(*args, **kwargs)
-        self.page_count = 0
-        self.page_size_query_param = None
+        self.page_count = None
+        self.pagination_page_size = None
 
     def paginate_queryset(self, queryset, request, view=None):
-        self.page_count = Subsidy.objects.count()
-        self.page_size_query_param = request.query_params.get('page_size') or self.page_size
+        if request.query_params.get('page_size'):
+            self.pagination_page_size = int(request.query_params.get('page_size'))
+        else:
+            self.pagination_page_size = self.page_size
```

Comment:
nit: you can do this as one statement in python with something like
```python
self.pagination_page_size = int(request.query_params.get('page_size', self.page_size))
```
Where `self.page_size` is the default value returned by `.get()` in case the `page_size` key isn't present in the query_params dict.